public/app/features/dashboard-scene/scene/PanelRepeaterGridItem.tsx

```diff
@@ -94,11 +95,12 @@ export class PanelRepeaterGridItem extends SceneObjectBase<PanelRepeaterGridItem
       return;
     }
 
-    const panelToRepeat = this.state.source;
+    let panelToRepeat =
+      this.state.source instanceof LibraryVizPanel ? this.state.source.state.panel! : this.state.source;
```

Comment:
@kaydelaney not sure I understand this, if we we only repeat the inner VizPanel the LibraryPanel  will never activate and load the library panel

Reply:
The LibraryPanel is already activated when we change its parent for a PanelRepeaterGridItem: https://github.com/grafana/grafana/pull/83579/files#diff-7539258415a1aa1738ddcc75b282954d692bf3d5774cce06e65194baecfc66dfR89